hs/WebParsing/TimeTableParser.hs

```diff
@@ -17,15 +18,84 @@ import WebParsing.TimeConverter
 import Control.Monad.IO.Class (liftIO, MonadIO)
 import Control.Monad.Trans.Reader (ReaderT)
 import Config (databasePath, timetableUrl)
+import Database.Tables (Department(Department))
```

Comment:
Don't repeat the import here (see line 13). You should just use `Tables.Department` in your code.